features/dev/persoon-adressering-inonderzoek.feature

```diff
@@ -0,0 +1,486 @@
+      | inOnderzoek.datumIngangOnderzoekVerblijfplaats.datum       | 2002-07-01                 |
+      | inOnderzoek.datumIngangOnderzoekVerblijfplaats.langFormaat | 1 juli 2002                |
+
+      Voorbeelden:
+      | gba in onderzoek waarde | adresregel1 in onderzoek | adresregel2 in onderzoek | adresregel3 in onderzoek | land in onderzoek |
+      | 080000                  | true                     | true                     | true                     | true              |
+      | 080900                  |                          | true                     |                          |                   |
+      | 080910                  |                          | true                     |                          |                   |
+      | 081100                  | true                     | true                     |                          |                   |
+      | 081110                  | true                     |                          |                          |                   |
+      | 081115                  | true                     |                          |                          |                   |
+      | 081120                  | true                     |                          |                          |                   |
+      | 081130                  | true                     |                          |                          |                   |
+      | 081140                  | true                     |                          |                          |                   |
+      | 081150                  | true                     |                          |                          |                   |
+      | 081160                  |                          | true                     |                          |                   |
+      | 081170                  |                          | true                     |                          |                   |
+      | 081200                  | true                     |                          |                          |                   |
+      | 081210                  | true                     |                          |                          |                   |
+      | 081300                  | true                     | true                     | true                     | true              |
+      | 081310                  |                          |                          |                          | true              |
+      | 081330                  | true                     |                          |                          |                   |
+      | 081340                  |                          | true                     |                          |                   |
+      | 081350                  |                          |                          | true                     |                   |
```

Comment:
Wordt het niet levberen vanb een element in een dergelijke tabel niet aangegeven met een "-" ? Dat heb ik op andere plekken wel gezien, of gebruiken we daarvoor zowel een lege tabel-entry als een "-". (En als het leeg moet zijn heb ik nog wat huiswerk in de historie-features.

Reply:
Die ben ik niet tegengekomen. Mijn voorkeur is geen '-' te gebruiken voor lege/geen waarde

Reply:
OK, Pas ik dat aan in de historie-features